similarity_api/src/settings.py

```diff
@@ -0,0 +1,32 @@
+import os
+import json
+
+def get_postgres_credentials_from_vcap():
+    vcap_services = os.environ.get('VCAP_SERVICES')
+
+    if vcap_services:
+        services = json.loads(vcap_services)
+
+        if 'aws-rds' in services:
+            postgres_service = services['aws-rds'][0]
+            return postgres_service['credentials']
+
+    return None
+
+vcap_credentials = get_postgres_credentials_from_vcap()
+
+DB_USERNAME = os.environ.get("POSTGRES_USERNAME") or (vcap_credentials and vcap_credentials.get('username'))
+DB_PASSWORD = os.environ.get("POSTGRES_PASSWORD") or (vcap_credentials and vcap_credentials.get('password'))
+DB_HOST = os.environ.get("POSTGRES_HOST") or (vcap_credentials and vcap_credentials.get('host'))
+DB_NAME = os.environ.get("POSTGRES_DB") or (vcap_credentials and vcap_credentials.get('db_name'))
+PORT = (vcap_credentials and vcap_credentials.get('port')) or 5432
+
+print("DB_USERNAME: {}".format(DB_USERNAME))
+print("DB_HOST: {}".format(DB_HOST))
+print("DB_NAME: {}".format(DB_NAME))
+print("PORT: {}".format(PORT))
```

Comment:
```suggestion

```

Reply:
I believe this can now be removed. This needs to be done for prod and non-prod envs since we don't want these values in the logs.

Reply:
23b837eb0